travis/publish.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -e
+
+# Setup login
+openssl aes-256-cbc -K $encrypted_fb15e83f0f9f_key -iv $encrypted_fb15e83f0f9f_iv \
+    -in .dockercfg.enc -out ~/.dockercfg -d
+
+
+JDK="oraclejdk7"   # what JDK we use to build the WAR that goes in the docker image
+NAME="camptocamp/mapfish_print"
+
+function publish {
+    local version=$1
+    export DOCKER_VERSION=`echo "${version}" | sed -e 's/^release\///' | sed -e 's/\//_/g'`
+
+    echo "Deploying image to docker hub for tag ${DOCKER_VERSION}"
+    ./gradlew --console=plain createDocker
+    docker push "${NAME}:${DOCKER_VERSION}"
+}
+
+if [ "${TRAVIS_JDK_VERSION}" != "${JDK}" ]
+then
+    echo "Only publishing with ${JDK}"
+    exit 0
+fi
+
+if [ "${TRAVIS_PULL_REQUEST}" == "true" ]
+then
+    echo "Not deploying image for pull requests"
+    exit 0
+fi
+
+if [ "${TRAVIS_BRANCH}" == "development" ] || [ "${TRAVIS_BRANCH}" == "docker" ]
+then
+  publish latest
```

Comment:
The latest tag is mainly for testing the last development version. If they really want to know what version they are running they can look at the manifest. People caring about what version they are running (the majority) should use a tagged version.

I don't know... you want me to remove the specification of DOCKER_VERSION for master and you'd get tags like "3.2.0-5-g51aff4e". To me that would be weird and we would have a confusing amount of tags that are not production ones.